Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/largefile/TimestampValidator.java

```diff
@@ -117,10 +122,10 @@ protected long getTimeStampTTL() {
     protected long getTimeStampFutureTTL() {
         try {
             return PropertyAccessor.getInstance().getPropertyLong(NhincConstants.GATEWAY_PROPERTY_FILE,
-                    NhincConstants.TIMESTAMP_FUTURE_TIME_TO_LIVE);
+                NhincConstants.TIMESTAMP_FUTURE_TIME_TO_LIVE);
         } catch (PropertyAccessException pae) {
             LOG.warn("Failed to determine timestamp future TTL in gateway.properties, will use default value: {}",
```

Comment:
consider to use warn or trace. It doesn't make sense to log twice with different message